Source/Core/Common/MathUtil.h

```diff
@@ -135,9 +135,13 @@ class RunningVariance
 
   constexpr size_t Count() const { return m_running_mean.Count(); }
   constexpr T Mean() const { return m_running_mean.Mean(); }
+
   constexpr T Variance() const { return m_variance / (Count() - 1); }
   constexpr T StandardDeviation() const { return std::sqrt(Variance()); }
```

Comment:
I'm pretty sure both this and `PopulationStandardDeviation` cannot ever actually be used in a constexpr context, since `std::sqrt` isn't constexpr

Reply:
True. I was hoping it would be `constexpr` in C++20, but apparently not.

Reply:
Fixed